pkg/cmd/pkg/testdata/describe.expected.json

```diff
@@ -0,0 +1,37 @@
+{
+  "package": {
+    "command": {},
+    "config": {
+      "foo": "bar"
+    },
+    "description": "Package stub with only required fields",
+    "maintainer": "John Doe",
+    "marathon": {
+      "v2AppMustacheTemplate": "ewogICAgImlkIjogInt7aGVsbG93b3JsZC5pZH19IiwKICAgICJjbWQiOiAiZWNobyAnSGVsbG8gV29ybGQnIgp9"
+    },
+    "name": "helloworld",
+    "packagingVersion": "0.0.1",
+    "releaseVersion": 9223372036854775807,
```

Comment:
I hope it's not the real value we get when calling Cosmos :sweat_smile:

Reply:
It's there to check for int32 overflow